eng/docker-compose/start-local-dms.ps1

```diff
@@ -60,20 +58,25 @@ if ($d) {
     }
 }
 else {
-    $upArg = ""
-    if ($r) { $upArg = "--build" }
+    $upArgs = @(
+        "--detach"
+    )
+    if ($r) { $upArgs += @("--build") }
```

Comment:
The change I made earlier fixed the situation I had locally, wanting to run the script with `-r`. But it broke the script when run _without_ `-r`. It turns out that `$upArg = ""` caused the `docker compose .. up $upArg` command to fail. Docker Compose was getting the empty string and responding with error. Which I still don't get. Weird. Changing to this array approach - which is what I had replaced earlier, though even that was not implemented quite right - fixed the core E2E test failure that began with my initial PR.